src/reconcile/rec_visibility.c

```diff
@@ -357,6 +360,7 @@ __rec_validate_upd_chain(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_UPDATE *s
             prev_upd->durable_ts >= vpack->tw.durable_stop_ts);
         if (prev_upd->start_ts < vpack->tw.start_ts ||
           (WT_TIME_WINDOW_HAS_STOP(&vpack->tw) && prev_upd->start_ts < vpack->tw.stop_ts)) {
+            WT_ASSERT(session, prev_upd->start_ts == WT_TS_NONE);
```

Comment:
I would have expected `cache_eviction_blocked_ooo_checkpoint_XXX` to go away entirely, is that not the case?

Reply:
No, we still need those scenarios because we don't want to evict a page that contains a mixed-mode update/tombstone also. So I changed the existing stats for the `mm`.